<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Order Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(203, 235, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stockTab">
     <attribute name="title">
      <string>Stock</string>
     </attribute>
     <widget class="QTableWidget" name="stockTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>432</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>List Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="refreshStockButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStockButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="customerTab">
     <attribute name="title">
      <string>Customers</string>
     </attribute>
     <widget class="QTableWidget" name="customerTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>431</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Discount</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="refreshCustomersButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addCustomersButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ordersTab">
     <attribute name="title">
      <string>Orders</string>
     </attribute>
     <widget class="QTableWidget" name="headersTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>431</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Customer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delivery Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delivery Charge</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Cost</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="linesTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>431</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>Book</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Line Price</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="linesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Order Lines:</string>
      </property>
     </widget>
     <widget class="QLabel" name="headersLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Order Headers:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addOrdersButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshOrdersButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(26, 140, 255);
font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QWidget" name="helpWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QTextBrowser" name="helpText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:10pt;&quot;&gt;- Use the tabs in the top left to navigate the different tables&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Calibri'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:10pt;&quot;&gt;- Press each coloumn Header to sort by that catagory&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Calibri'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:10pt;&quot;&gt;- Click the Add button on each tab to add new data into the table&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitHelpButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Calibri&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.0223464, y2:1, stop:0 rgba(153, 0, 0, 255), stop:1 rgba(255, 67, 67, 255));</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOrders">
   <property name="text">
    <string>Orders</string>
   </property>
  </action>
  <action name="stockAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="stockSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="stockSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="customerAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="stockUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="customerUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="customerSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="customerSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="orderAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="orderUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="orderSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="orderSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="actionReadData">
   <property name="text">
    <string>ReadData</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>refreshCustomersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshCustomers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCustomersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addCustomer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshStockButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshStock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addStockButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addStock()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshOrdersButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshOrders()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitHelpButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshCustomers()</slot>
  <slot>addCustomer()</slot>
  <slot>refreshStock()</slot>
  <slot>addStock()</slot>
  <slot>refreshOrders()</slot>
  <slot>help()</slot>
  <slot>closeHelp()</slot>
 </slots>
</ui>
